SPEC.md

```diff
@@ -413,6 +605,7 @@ space](https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt):
 | Name                 | Code Pt |
 |----------------------|---------|
 | Character Tabulation | `U+0009`  |
+| Line Tabulation      | `U+000B`  |
```

Comment:
This has also been added as a Disallowed Literal Code Point right?

Reply:
And U+0009 too.

Reply:
wait, this seems like an error. Why did we disallow literal tabs?...

Reply:
491cc46f89df0261b494ccc89c2bc7b243467bb2

Reply:
Well, I guess I just got the range wrong. It should be 0x00-0x08. The rest are fine, I think.

Reply:
0x0B should be removed from the range too, I think.